src/main/resources/META-INF/plugin.xml

```diff
@@ -0,0 +1,24 @@
+<idea-plugin>
+    <id>org.example.mdx-intellij-plugin</id>
```

Comment:
mdx.js

Reply:
Changed id to mdx.js, but I am not sure that I understood you right